.github/actions/rdp/action.yaml

```diff
@@ -0,0 +1,36 @@
+name: Azure RDP setup
+description: 'Connect via RDP in order to log in as session other than 0.'
+
+inputs:
+  az_vm_hostname:
+    description: Hostname for "xfreerdp /v:az_vm_hostname"
+    required: true
+  az_vm_username:
+    description: Username for "xfreerdp /u:az_vm_username"
+    required: true
+  az_vm_password:
+    description: Password for "xfreerdp /p:az_vm_password"
+    required: true
+
+runs:
+  using: 'composite'
+  steps:
+    - name: rdp
+      shell: bash
+      run: |
+        set -o pipefail
+        sudo apt-get install -y xvfb freerdp2-x11
```

Comment:
You need to set that it’s non interactive to avoid debconf prompts (`DEBIAN_FRONTEND=noninteractive`)

Reply:
DEBIAN_FRONTEND=noninteractive still missing.